svo (custom prompt, serverside).xml

```diff
@@ -813,8 +813,12 @@ signals["svo system loaded"]:connect(function()
 end, 'get initial state of prios')
 
 function sk.resetserversideprios()
-  svo.sendc("curing priority reset")
+  --[[svo.sendc("curing priority reset")
```

Comment:
Remove all of the commented out things if they're no longer needed, the code will get hard to read with them otherwise. We can always pull them out of Git's history if needed.